lib/valid_tags.rb

```diff
@@ -0,0 +1,79 @@
+class ValidTags
+  ALLOWED_TAGS = %i(
```

Comment:
Do we need to update https://docs.publishing.service.gov.uk/apis/search/add-new-fields-or-document-types.html as well to mention updating this list?

Reply:
It sounds like that doc concerns search fields, though I think there's a difference between fields that one can search with and fields accepted by email alert api.

I couldn't find a doc on adding email alerts for a finder, so it might be worth adding one here: https://github.com/alphagov/finder-frontend/tree/master/docs, and then I can link to this doc from there. What do you think?

Reply:
Yeah, I reckon that's probably fine. I'm imaging a situation where someone adds a new field that should also be supported by email-alert-api and they have no idea about the whitelist and get confused wondering why it's not working. So as long as it's written down somewhere.

Reply:
I added documentation for this step to finder-frontend: https://github.com/alphagov/finder-frontend/commit/7ac871987abf36184b136b3b3513ebaa02d8e8a6#diff-b94f09a9b9baf14806b87c9db6ae7d0eR93